ClearKeep/Scenes/Home/Views/ServerView.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  ServerView.swift
+//  ClearKeep
+//
+//  Created by NamNH on 01/03/2022.
+//
+
+import SwiftUI
+
+struct ServerView: View {
+	
+	private var items: [String] = ["a", "b", "c"]
+	
+    var body: some View {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/3e6c3ff22fa5861f78566849d80733b6d3d53536/ClearKeep/Scenes/Home/Views/ServerView.swift#L14"></span>Please use tabs instead of spaces<br />
<code>not_spaces</code> <code>ServerView.swift:14</code></td>
    </tr>
  </tbody>
</table>